### PR TITLE
[testing] Bug fix/asan timeouts, disable coredumping

### PR DESCRIPTION
**Pull request by dothebart**
### Scope & Purpose

- properly set timeout for *san tests
- properly handle crashing tests

- [x] :hankey: Bugfix
- [x] backports
  - [x] 3.10 https://github.com/arangodb/arangodb/pull/17508
  - [x] 3.9 https://github.com/arangodb/arangodb/pull/17509
  - [x] 3.8 https://github.com/arangodb/arangodb/pull/17511

### CODE REVIEW

File: js/client/modules/@arangodb/testutils/testing.js [hunk 1]
```diff
@@ -214,10 +218,8 @@ const optionsDefaults = {
   'skipNondeterministic': false,
   'skipGrey': false,
   'onlyGrey': false,
-  'oneTestTimeout': 15 * 60,
-  'isAsan': (
-      global.ARANGODB_CLIENT_VERSION(true).asan === 'true' ||
-      global.ARANGODB_CLIENT_VERSION(true).tsan === 'true'),
+  'oneTestTimeout': (isAsan? 15 : 25) * 60,
```
Comment (jsteemann) [inline js/client/modules/@arangodb/testutils/testing.js:221]: the timeout is now lower with ASan than without? Does this make sense?

**Comment by dothebart** [comment]
https://jenkins01.arangodb.biz/job/arangodb-ANY-linux-san.x86-64/1330/

**Comment by KVS85** [comment]
This must be done in oskar via system settings at https://github.com/arangodb/oskar/pull/442.

**Comment by jsteemann** [comment]
https://jenkins.arangodb.biz/view/PR/job/arangodb-matrix-pr/25395/

**Review by KVS85 — APPROVED** [review]

**Review by jsteemann — APPROVED** [review]
LGTM

**Comment by jsteemann** [comment]
https://jenkins.arangodb.biz/view/PR/job/arangodb-matrix-pr/25414/

**Pull request merged**